vmngclient/session.py

```diff
@@ -19,9 +21,56 @@
 logger = logging.getLogger(get_logger_name(__name__))
 
 
+def Create(
```

Comment:
i would use create (not capitalised - using standard snake case convention for python methods)

Reply:
It is done in like in [Netmiko](https://github.com/ktbyers/netmiko) which uses similar factory function [here](https://github.com/ktbyers/netmiko/blob/develop/netmiko/ssh_dispatcher.py#L358).

The end result is that when using `Create` function to create it looks more user friendly with camel case because it returns a session object, even though it is not standard for python.
```python
# camel case
session = Create(url, port, username, password)

# snake case
session = create(url, port, username, password)
```

Reply:
named `create_session`